techniques/failures/F46.html

```diff
@@ -70,8 +70,9 @@
                                     table element</li>
             <li>
                <p>If the table is used only to visually lay out elements within the
-                                    content </p>
+                                    content:</p>
                <ol>
+                  <li>Check that the table is marked presentational by methods such as role none or presentation on the table element.</li>
```

Comment:
I'm just wondering if we should be clear that we're talking about role="none" and role="presentation".

It seems that the non-technical approach is both more confusing and more lengthy. Is there a reason not to be specific?

Reply:
I think that changing the text to reflect how the actual tag appears is the correct thing to do here.